app/src/main/res/values/dimens.xml

```diff
@@ -9,6 +9,9 @@
     <dimen name="margin_normal">16dp</dimen>
     <dimen name="margin_medium">24dp</dimen>
     <dimen name="margin_big">32dp</dimen>
+    <dimen name="padding_small">10dp</dimen>
+    <dimen name="cardView_dimensions_about_act">120dp</dimen>
     <dimen name="test_size_title">18sp</dimen>
     <dimen name="test_size_title_big">24sp</dimen>
+    <dimen name="ico_size_about_actvt">50dp</dimen>
```

Comment:
@MariamKotob the variable names seems a bit odd, can you use simpler names like `value_50dp` etc

Reply:
ok will make it.